App/Sources/DesignSystem/Extensions/View/MainRankingView.swift

```diff
@@ -0,0 +1,73 @@
+            ScrollView(.horizontal, showsIndicators: false) {
+                HStack(spacing: 15.44) {
+                    ForEach(1..<4) { i in
+                        ZStack(alignment: .topLeading) {
+                            VStack {
+                                Image(.bannerImage3)
+                                    .resizable()
+                                    .aspectRatio(contentMode: .fill)
+                                    .frame(width: 60, height: 60)
+                                    .clipShape(Circle())
+                                    .clipped()
+                                
+                                Text(profileName)
+                                    .skFont(.pm14)
+                                    .lineLimit(nil)
+                                    .fixedSize(horizontal: false, vertical: true)
+                                
+                                Spacer().frame(height: 16)
+                                
+                                Text(ownPoint)
+                                    .skFont(.pm16)
+                            }
+                            .padding(.init(top: 18, leading: 25, bottom: 18, trailing: 25))
+                            .background(Color.white)
+                            .frame(height: 158)
+                            .cornerRadius(10)
+                            
+                            if i == 1 {
+                                Circle()
+                                    .frame(width: 20, height: 20)
+                                    .foregroundColor(Color.SKColorSystem.Medal.firstplace.color)
+                                    .overlay(
+                                        Text("1")
+                                            .skFont(.pm14)
+                                    )
+                            }
+                            else if i == 2 {
+                                Circle()
+                                    .frame(width: 20, height: 20)
+                                    .foregroundColor(Color.SKColorSystem.Medal.secondplace.color)
+                                    .overlay(
+                                        Text("2")
+                                            .skFont(.pm14)
+                                    )
+                            }
+                            else if i == 3 {
+                                Circle()
+                                    .frame(width: 20, height: 20)
+                                    .foregroundColor(Color.SKColorSystem.Medal.thirdplace.color)
+                                    .overlay(
+                                        Text("3")
+                                            .skFont(.pm14)
+                                    )
+                            }
+                        }
+                    }
+                }
+            }
+            .padding(.vertical, 16)
+            .padding(.horizontal, 15.44)
```

Comment:
```suggestion
            .padding(.horizontal, 16)
```
소수점보다 4의 배수로 수정하는 게 좋을 것 같아요!

Reply:
소수점 다 고친줄 알았는데 피차 확인못한 게 있었네요... 알겠습니다~!